libs/tup-components/src/tickets/TicketCreateForm.global.css

```diff
@@ -0,0 +1,63 @@
+.ticket-create-form {
+  display: flex;
+  flex-direction: column;
+  font-size: 0.75rem;
+ /* Override Bootstrap `.form-text` (being used in _common/Form/FormField.js), because we need the field text to be the same size as the field label */
+ /* alter bootstrap size to match font-size for rest of ticket forms */
+}
+
+.ticket-create-form .form-text {
+  font-size: 0.75rem;
+}
+
+.ticket-create-form .col-form-label-sm {
+  font-size: 0.75rem;
+}
+
+.ticket-description-text-area {
+  min-height: 5em;
+}
+
+.ticket-create-button-row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: right;
+}
+
+.ticket-create-spinner {
+  margin-right: 0.5em;
+}
+
+.ticket-create-info-alert {
+  margin-right: 1em;
+}
+
+.ticket-create-button-row .alert {
+  margin-bottom: 0;
+  border: none;
+}
+
+.ticket-create-button-row .btn-primary {
+  margin-left: auto;
+  border: none;
+  background-color: var(--global-color-accent--normal);
+  font-size: 0.75rem;
+}
+
+.ticket-link {
+  color: var(--global-color-accent--normal);
+}
+
+.form-validation-error {
+  color: #eb6e6e;
+}
```

Comment:
**Forenotice:** We have `.c-form__errors` ([c-form](http://localhost:8000/static/ui/components/detail/c-form.html)) which will style errors. So, this might go away.

Reply:
Yes, this became unnecessary; it has been removed.